spec/models/user_spec.rb

```diff
@@ -90,6 +90,19 @@
         expect(user.errors.first.details).to eq({ error: :confirmation, attribute: 'Password' })
       end
     end
+
+    context 'email format' do
+      it 'accepts valid email format' do
+        user = build(:user, email: 'user@users.com')
+        expect(user).to be_valid
+      end
+
+      it 'refuses invalid email formats' do
+        user = build(:user, email: 'INVALID')
+        expect(user).to be_invalid
+        expect(user.errors.attribute_names).to match_array([:email])
```

Comment:
The matching instead of inclusion is intentional to ensure that only the email attribute is invalid,